api/policies.kyverno.io/v1alpha1/validating_policy.go

```diff
@@ -191,7 +192,7 @@ type ValidatingPolicySpec struct {
 	// ValidationAction specifies the action to be taken when the matched resource violates the policy.
 	// Required.
 	// +listType=set
-	ValidationAction []admissionregistrationv1.ValidationAction `json:"validationActions,omitempty"`
```

Comment:
Is this really required ? There's no defaulting ?

Reply:
Nope. It can be unset, and the policy can be created without it, which shouldn't be. It is also required in the VAP Binding.

Reply:
Yes, we already assume it is `Deny` by default:
https://github.com/kyverno/kyverno/blob/af8d3946b444f290585b666b5755af341a663223/pkg/cel/engine/provider.go#L75-L78

But we have to check the values of the validation actions if they are set because currently, if the user sets it to `test`, the policy will be accepted, which shouldn't be.

For example, this policy will be successfully created.
```
apiVersion: policies.kyverno.io/v1alpha1
kind: ValidatingPolicy
metadata:
  name: disallow-privilege-escalation-1
spec:
  autogen:
    validatingAdmissionPolicy:
      enabled: true
  validationActions: ["testtt"]
```

Reply:
We also have to document that the default is `Deny` if this field is unset because we aren't using a Kubebuilder marker to define this default directly in the schema.